package.json

```diff
@@ -58,8 +51,18 @@
     ],
     "failFast": true,
     "require": [
-      "babel-register"
+      "@babel/register",
+      "@babel/polyfill"
     ],
-    "babel": "inherit"
+    "babel": {
+      "testOptions": {
+        "presets": [
+          "@babel/preset-env"
+        ],
+        "plugins": [
+          "@babel/plugin-transform-async-to-generator"
+        ]
+      }
+    }
```

Comment:
Doesn't `ava`'s babel config read the regular Babel config? Having it duplicated isn't particularly maintainable.